File: Java/Tests/jdi-uitest-cucumbertests/src/main/java/com/epam/jdi/uitests/testing/unittests/pageobjects/pages/HomePage.java
package com.epam.jdi.uitests.testing.unittests.pageobjects.pages;

import com.epam.jdi.uitests.core.interfaces.common.IImage;
import com.epam.jdi.uitests.web.selenium.elements.common.Text;
import com.epam.jdi.uitests.web.selenium.elements.composite.WebPage;
import org.openqa.selenium.support.FindBy;

/**
 * Created by Maksim_Palchevskii on 8/17/2015.
 */

public class HomePage extends WebPage {
    @FindBy(css = ".main-txt")
    public Text text;
    @FindBy(css = ".epam-logo img")
    public IImage logoImage;
}

File: Python/Test/jdi_uitests_webtests/main/page_objects/pages/home_page.py
from JDI.web.selenium.elements.common.image import Image
from JDI.web.selenium.elements.common.text import Text
from JDI.web.selenium.elements.composite.web_page import WebPage
from JDI.web.selenium.elements.api_interact.find_element_by import By
from JDI.web.selenium.elements.common.link import Link


class HomePage(WebPage):

    def __init__(self, url, title):
        super(HomePage, self).__init__(url=url, title=title)

    about = Link(By.link_text("About"))

    logo_image = Image(By.css(".epam-logo img"))

    text_item = Text(By.css(".main-txt"))